### ISSUE TITLE
Tagline

### ISSUE THREAD

**Issue by sindresorhus** [imagemin/meta#4]
Currently: "Minify images seamlessly"

I think we need something more memorable.

Suggestions:
- Optimized for happiness
- Makes everything better
- For a faster web
- Minifies images like there's no tomorrow
- Why Have Cotton When You Can Have Imagemin?
- You've Got Bloated Images. We've Got Imagemin.
- Minifies images so you don't have to
- Tense, Nervous, Minifying Images?
- Behold the Power of Imagemin

**Comment by kevva** [comment]
haha, some of these are brilliant.

**Comment by kevva** [comment]
- Just _minify_ it
- Wake up and _minify_
- Did somebody say Imagemin?
- Because your images are worth it
- Look, Ma, no bloated images!
- Easy as Imagemin
- We bring images to life
- Think small

these are all taglines we can't use.

**Comment by sindresorhus** [comment]
I like `Think small`

**Comment by kevva** [comment]
You think Volkswagen will notice?

**Comment by sindresorhus** [comment]
Oh, I thought it was a pun on Apple's `Think different`. It's fine either way though.

**Comment by kevva** [comment]
Yep, that one's good. Or instead of `We bring images to life`, `We bring images to live`, hehehe.

**Comment by sindresorhus** [comment]
I kinda like `Optimized for happiness` too.

**Comment by kevva** [comment]
Indeed, we got some good ones to choose from here. `Tense, Nervous, Minifying Images?` is so funny, lol.

**Comment by sindresorhus** [comment]
Indeed :)

I'll do a tweet about this thread later. We could use some outside feedback.

**Comment by razvanz** [comment]
_Why only JS and CSS?_
but I would prefer:
_Squash only the best._

**Comment by thasmo** [comment]
Min. Imagemin. Optimized not Crushed.
Less image, more min.
Min to win!

I tried ... :>

**Comment by CWSpear** [comment]
I really liked `Tense, Nervous, Minifying Images?`

**Comment by jeanbza** [comment]
I also liked `Think small`. Here are some others:

`Smaller is better`
`Same bang, smaller buck`
`Go small or go home`
`Like, tiny`
`Ain't nobody got time for huge images`

**Comment by smaye81** [comment]
Size does matter

**Comment by kevva** [comment]
> Size does matter

That's a good one!

**Comment by sindresorhus** [comment]
:+1:
